Dockerfile

```
@@ -1,14 +1,10 @@
 FROM node:20-alpine3.17
 
-RUN apk add --no-cache python3 make g++  \
+RUN apk add --no-cache python3 make g++ curl  \
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)